clang-tools-extra/clang-tidy/ClangTidy.cpp

```diff
@@ -189,9 +191,12 @@ class ErrorReporter {
 
   void finish() {
     if (TotalFixes > 0) {
-      Rewriter Rewrite(SourceMgr, LangOpts);
+      bool AnyNotWritten = false;
       for (const auto &FileAndReplacements : FileReplacements) {
+        Rewriter Rewrite(SourceMgr, LangOpts);
         StringRef File = FileAndReplacements.first();
+        Files.getVirtualFileSystem().setCurrentWorkingDirectory(
```

Comment:
maybe would be nice to restore CurrentWorkingDirectory after this function come to the end.